src/protocol.h

```diff
@@ -136,25 +136,43 @@ class CInv
 public:
     CInv();
     CInv(int typeIn, const uint256& hashIn);
+    CInv(int typeIn, const uint256& hashIn, const uint256& hashAuxIn);
 
     ADD_SERIALIZE_METHODS;
 
     template <typename Stream, typename Operation>
     inline void SerializationOp(Stream& s, Operation ser_action)
     {
+        int nVersion = s.GetVersion();
         READWRITE(type);
         READWRITE(hash);
+        if (type == MSG_WTX && nVersion >= CINV_WTX_VERSION) {
+            // We've already checked above that nVersion >= CINV_WTX_VERSION.
```

Comment:
```suggestion
```
Comment doesn't seem helpful when it's literally the line above.

Reply:
Gah, this also slipped in, and should have been part of the later commit. I'll refactor these to be correct.